src/AudacityApp.cpp

```diff
@@ -1149,6 +1149,8 @@ bool AudacityApp::OnInit()
 
    wxString progPath = wxPathOnly(argv[0]);
    FileNames::AddUniquePathToPathList(progPath, audacityPathList);
+   // Add the path to modules:
+   FileNames::AddUniquePathToPathList(progPath + L"/lib/audacity", audacityPathList);
```

Comment:
Is this correct for installed versions? In that case Audacity would be `/usr/bin/audacity`, but the modules would be `/usr/lib/audacity`, and I think this code would give `/usr/bin/lib/audacity` if I am reading it correctly.